src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_workload_commands.py

```diff
@@ -561,13 +561,144 @@ def test_backup_wl_sql_restore_as_files(self):
 
         self.cmd('backup job wait -v {vault} -g {rg} -n {job}')
 
+    @record_only()
+    def test_workload_archive_sql (self):
```

Comment:
Change the test name to test_backup_wl_sql_archive

Reply:
Done